WebRoot/js/zimbraMail/briefcase/view/ZmPreviewPaneView.js

```diff
@@ -64,7 +64,21 @@ function() {
 
 ZmPreviewPaneView.SASH_THRESHOLD = 5;
 ZmPreviewPaneView._TAG_IMG = "TI";
+ZmPreviewPaneView.REMOVE_FILE_BUTTON_ID = "ZmPreviewPaneViewButton";
 
+ZmPreviewPaneView.getHTML = function (){
```

Comment:
```suggestion
ZmPreviewPaneView.getHTML =
function() {
```

Reply:
after moving the html code to template files this function is not needed. Hence removed it.